Web/Scotty/Action.hs

```diff
@@ -132,6 +132,39 @@ actionErrorHandler = Handler $ \case
   AENext -> next
   AEFinish -> return ()
 
+-- | Default, unscented handler for exceptions from scotty
+scottyExceptionHandler :: MonadIO m => ErrorHandler m
+scottyExceptionHandler = Handler $ \case
+  RequestTooLarge -> do
+    status status413
+    text "Request body is too large"
+  MalformedJSON bs err -> do
+    status status400
+    text $ T.unlines
+      [ "jsonData: malformed"
+      , "Body: " <> decodeUtf8Lenient (BL.toStrict bs)
```

Comment:
do we really need to do a full copy and decode here?

Reply:
Is your suggestion to decode lazily?

Reply:
I switched to use `raw` instead